contributing-guides/style-guide.md

```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
 
 The basic format of each page should match the following template:
 
-```
-# command-name
+```md
+# <command-name>
```

Comment:
I don't like this, people will assume it needs to be wrapped in them, leading to just more linting and syntax fixing.

Reply:
Fixed: angle brackets are removed.

Reply:
Angle braces are just removed.